### PR TITLE
option 1: big merge with all the bluemix and watson commit history (merge from bluemix+watson to here)

### PR DESCRIPTION
**Pull request by ricarkol**
This merge includes all changes from watson and alchemy repos while preserving their commit history. The trick used to solve the issue with not having a common ancestor was to force one: `git replace --graft 4624c5011d8b13d77572626157b61863393ea94d e0335711669ada948fde17df4079070e98cd97c7`. With this, commit e033571 in the agentless-system-crawler repo looks like the ancestor of commit 4624c501 in the watson+bluemix repo. After that, just did a plain merge and fixed the conflicts. `git rebase` was also OK, but i couldn't find a way of keeping the commit history.

Tested with all the *.sh tests and the py.test ones.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<<<<<<< HEAD
+.vagrant/*
+*.pyc
+=======
```
Comment (MHBauer) [inline .gitignore]: unresolved merge conflicts in here.

File: LICENSE [hunk 2]
```diff
@@ -1,177 +0,0 @@
-
```
Comment (MHBauer) [inline LICENSE]: I don't see a replacement license file for people to know what license this is.

**Comment by duglin** [comment]
when I build this with `make` I get:
```
Step 5 : COPY crawler/requirements.txt /requirements.txt
 ---> 37cb950c1b21
Removing intermediate container 184eb259159b
Step 6 : COPY tests/requirements.txt /requirements-test.txt
lstat tests/requirements.txt: no such file or directory
Makefile:9: recipe for target 'test' failed
make: *** [test] Error 1
```
do you see this too?

**Comment by MHBauer** [comment]
same as @duglin. no requirements.txt file present in the tests directory.

maybe another push of updates is needed @ricarkol ?

**Comment by duglin** [comment]
LGTM
just built and tested and it looks good

**Comment by ricarkol** [comment]
Just pushed, thanks for the reviews!

**Pull request merged**